book/modules/appendix1-exercises.tex

```diff
@@ -52,59 +94,253 @@
 				If so, how many? Justify your answer.
 		\end{enumerate}
 
-		\prob On Kokoro's farm, there is a cage with $35$ animals, some of which 
-		are chickens
-		and some of which are rabbits. Kokoro counted the total number of legs in the
-		cage and found that there were $94$ legs in all (notably, each chicken
-		has exactly two legs and each rabbit has four legs). Kokoro decides
-		to use this information to figure out how many chickens and how many
-		rabbits there are\footnote{ This problem based on a
-		classical Chinese problem from the ancient Chinese treatise
-		\emph{Mathematical Classic of Master Sun} (or \emph{Sunzi Suanjing}) written
-		during 3rd to 5th centuries \textsc{a.d.}}.
+		\begin{solution}
+			\begin{enumerate}
+				\item If $\vec b=\mat{7 \\ 14}$, then the vector equation
+					becomes
+					\[
+						x\mat{2 \\ 4}+y\mat{8 \\ 16}=\mat{7 \\ 14}.
+					\]
+					Convert it to a system of linear equations:
```

Comment:
Converting it to a system of linear equations and row reducing we get `XXX -> YYY`